sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -39,6 +39,11 @@ internal static class Constants
         /// </summary>
         public const int LargeUploadThreshold = 100 * Constants.MB;
 
+        /// <summary>
+        /// The minimum number of bytes to download in Open Read.
```

Comment:
doc says "minimum", constant name "default" - that's confusing.
if it's default - isn't this too low?

Reply:
1 MB in this case.  We could change it to 4 MB.